java/ingest/ingest-core/src/main/java/sleeper/ingest/job/status/IngestJobStatusStore.java

```diff
@@ -44,10 +44,15 @@ default void jobStartedWithValidation(String taskId, IngestJob job, Instant star
     default void jobStarted(String taskId, IngestJob job, Instant startTime, boolean startOfRun) {
     }
 
+    default void jobStarted(String runId, String taskId, IngestJob job, Instant startTime, boolean startOfRun) {
+    }
 
     default void jobFinished(String taskId, IngestJob job, RecordsProcessedSummary summary) {
     }
 
+    default void jobFinished(String runId, String taskId, IngestJob job, RecordsProcessedSummary summary) {
+    }
```

Comment:
We could wrap the parameters in an object so that we've just got one method signature for each update type?